orchestra/project_api/tests/test_project_api.py

```diff
@@ -562,7 +574,7 @@ def test_get_single_todo_by_pk(self):
         self.assertEqual(resp.status_code, 200)
 
     def test_get_list_of_todos_with_filters(self):
-        url_with_project_filter = '{}?project={}'.format(
+        url_with_project_filter = '{}?project__id={}'.format(
```

Comment:
I know that we might end up with long method names but can we create separate tests for different scenarios e.g. when project_id is provided with/without id__in? I feel tests will be more readable that way. 

If you feel otherwise, can you add comments about each scenario right before you test it.

Reply:
Done :)